localenv/README.md

```diff
@@ -162,34 +157,36 @@ pnpm localenv:compose down --volumes
 
 ### Usage
 
-#### Postman & Open Payments APIs
+#### Bruno & Open Payments APIs
 
-The Open Payments APIs can be interacted with using the Postman collection ([Open Payments APIs](https://www.postman.com/interledger/workspace/interledger/folder/22855701-1b204bc1-c8e5-44d4-bab9-444d7204b15a?ctx=documentation) and [Open Payments Auth APIs](https://www.postman.com/interledger/workspace/interledger/folder/22855701-ae80b96d-4d25-42b9-94fa-8ed17f0e5ed9?ctx=documentation)). It is configured to use the default endpoints of the local environment.
+The Open Payments APIs can be interacted with using the [Bruno collection](https://github.com/interledger/rafiki/main/bruno/collections/Interledger) and its ([Open Payments APIs](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/Open%20Payments%20APIs) and [Open Payments Auth APIs](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/Open%20Payments%20Auth%20APIs)). It is configured to use the default endpoints of the local environment.
 
-The Examples folder in the Postman collection includes an [eCommerce example](https://www.postman.com/interledger/workspace/interledger/folder/22855701-e27838da-dd72-4b5e-9f1e-086ddfa4d098?ctx=documentation) that can be executed one by one. It
+The Examples folder in the Bruno collection includes an [eCommerce example](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/Examples/Open%20Payments) that can be executed one by one. It
```

Comment:
Maybe its worth explicitly stating the step to load the collection folder in bruno. Either somewhere here or `To use the Bruno collection examples follow the steps:`? I was thinking somewhere here since we are mentioning where its located and the steps to use it are more for using it, not setting it up.

The question popped into my head and was easily answered by thinking for half a second but I feel like it may not be so obvious for someone with 0 context. For example, with postman you didn't need to load anything, it just worked*

*<sub>except it actually didnt work <sub>